webpack.config.js

```diff
@@ -0,0 +1,26 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  entry: './src/index.js',
```

Comment:
-  [ ] OPTIONAL: I think it is better if you set the mode to 'development' here. Otherwise, it gives a warning that it will fall back to 'production' mode. It is excellent work though. :+1: :clap: 
![Screenshot from 2021-08-31 16-43-46](https://user-images.githubusercontent.com/64894927/131489341-19cf2c15-f826-47f5-b124-499b1710c8cc.png)